src/components/Details.js

```diff
@@ -1,9 +1,30 @@
 import React from 'react';
 
 export default class Details extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {message: 'Hello, this will be the details page for each Movie & TV show'};
+    }
+
+    componentDidMount() {
+        this.timerID = setInterval(
+            () => this.newMessage(),
```

Comment:
I'd be great to keep a more explicit, action-oriented naming conventions to describe functions:
1. Set the action you are going to perform (e.g. `setWhatever`, `returnSomething`, etc).
2. Describe the purpose of the action (you are setting a `Coming Soon` message, so the name of the action should reflect that, somehow.